src/main/java/network/brightspots/rcv/ContestConfig.java

```diff
@@ -70,6 +70,8 @@ class ContestConfig {
   private static final int MAX_ROW_INDEX = 100000;
   private static final int MIN_MAX_RANKINGS_ALLOWED = 1;
   private static final int MIN_MAX_SKIPPED_RANKS_ALLOWED = 0;
+  private static final int MIN_NUMBER_OF_ROUNDS = 0;
+  private static final int MAX_NUMBER_OF_ROUNDS = Integer.MAX_VALUE;
```

Comment:
Should MAX_NUMBER_OF_ROUNDS be here or on line 710 in the call to `fieldOutOfRangeOrNotInteger`?